.circleci/config.yml

```diff
@@ -11,12 +11,10 @@ _snyk_options: &snyk_options
 
 jobs:
   validate:
-    executor: hmpps/java
```

Comment:
this line pulls in the JVM configuration and other bits and pieces defined for the executor. would prefer to leave in.

Reply:
👍 Ah, I had a schema validator tell me it's one or the other, you're right.

TIL the expanded version gets an extra `environment` after disposing of the [original image](https://github.com/ministryofjustice/hmpps-circleci-orb/blob/aa63b0b076d564ba1b4e933fdda7f75065dfa75f/src/executors/java.yml) defined in the executor.
```
$ circleci config process .circleci/config.yml
<snip>
  validate:
    docker:
    - image: cimg/openjdk:11.0
      environment:
        POSTGRES_DB: circle_test
    - image: circleci/postgres:13-ram
      environment:
        POSTGRES_PASSWORD: password
<snip>
    environment:
    - _JAVA_OPTIONS: -Xmx256m -XX:ParallelGCThreads=2 -XX:ConcGCThreads=2 -XX:ParallelGCThreads=2 -Djava.util.concurrent.ForkJoinPool.common.parallelism=2 -Dorg.gradle.daemon=false -Dorg.gradle.jvmargs=-XX:+UseContainerSupport
```

It's back now.